feature/home/presentation/src/main/java/com/pob/umpa/feature/home/presentation/CommunityNoImage.kt

```diff
@@ -0,0 +1,179 @@
+package com.pob.umpa.feature.home.presentation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.pob.umpa.core.designsystem.R
+import com.pob.umpa.core.designsystem.theme.UmpaTheme
+
+data class Community(
+    val id: Long,
+    val title: String,
+    val createdAt: String,
+    val likeCount: Int,
+    val commentCount: Int,
+)
+
+private val dummyCommunityList = listOf(
+    Community(
+        id = 1,
+        title = "제 999회 현지와 함께하는 음파 경연대회 공모 요강 어서 빨리 ~~~",
+        createdAt = "2025.03.16 09:24",
+        likeCount = 10,
+        commentCount = 5,
+    ),
+    Community(
+        id = 1,
+        title = "제 999회 현지와 함께하는 음파 경연대회 공모 요강 어서 빨리 ~~~",
+        createdAt = "2025.03.16 09:24",
+        likeCount = 10,
+        commentCount = 5,
+    ),
+    Community(
+        id = 1,
+        title = "제 999회 현지와 함께하는 음파 경연대회 공모 요강 어서 빨리 ~~~",
+        createdAt = "2025.03.16 09:24",
+        likeCount = 10,
+        commentCount = 5,
+    ),
+)
```

Comment:
```suggestion
private val dummyCommunityList = listOf(
    Community(
        id = 1,
        title = "제 999회 현지와 함께하는 음파 경연대회 공모 요강",
        createdAt = "2025.03.16 09:24",
        likeCount = 10,
        commentCount = 5,
    ),
    Community(
        id = 2,
        title = "음파 마스터 클래스: 초보자를 위한 첫걸음",
        createdAt = "2025.03.17 14:35",
        likeCount = 25,
        commentCount = 12,
    ),
    Community(
        id = 3,
        title = "2025 음파 트렌드 예측 토론회 안내",
        createdAt = "2025.03.18 10:10",
        likeCount = 8,
        commentCount = 3,
    ),
    Community(
        id = 4,
        title = "음파 커뮤니티 정기 모임 후기 공유",
        createdAt = "2025.03.19 18:45",
        likeCount = 15,
        commentCount = 7,
    ),
    Community(
        id = 5,
        title = "최신 음파 기술 Q&A 세션 참가 신청",
        createdAt = "2025.03.20 13:20",
        likeCount = 30,
        commentCount = 20,
    ),
)
```
더미 데이터를 다양하게 추가 했읍니다..^^